src/components/ProfileDropdown.jsx

```diff
@@ -126,14 +101,21 @@ export default function ProfileDropdown(props) {
             <Typography component="h6" gutterBottom>
               {email}
             </Typography>
-            {display[0].manageProfile}
+            {EXPERIMENT_SHOW_FULL_PROFILE_MENU && (
+              <>
+                <a href={manageProfileLink}>Manage Your Profile</a>
+              </>
```

Comment:
The `<>` here is unnecessary. It only has **one** child element.

Reply:
Ok got that change in